src/Symfony/Component/HttpKernel/Log/DebugLoggerInterface.php

```diff
@@ -23,11 +23,15 @@ interface DebugLoggerInterface
     /**
      * Returns an array of logs.
      *
-     * A log is an array with the following mandatory keys:
-     * timestamp, message, priority, and priorityName.
-     * It can also have an optional context key containing an array.
-     *
-     * @return array
+     * @return array<array{
+     *     channel: ?string,
+     *     context: array<string, mixed>,
```

Comment:
Could be an empty array too

Reply:
Is there a specific annotation for empty arrays?

Reply:
An empty array totally satisfies `array<string, mixed>`